app/packages/utilities/src/index.ts

```diff
@@ -312,6 +314,15 @@ export const VALID_NUMERIC_TYPES = [
   INT_FIELD,
 ];
 
+// list fields may not have a subfield type, so null, undefined is included
+export const UNSUPPORTED_FILTER_TYPES = [
+  ARRAY_FIELD,
```

Comment:
Just making sure that array ARRAY_FIELD here does not exclude any custom array of primitive strings from being rendered as a regular field (as opposed to a field under `Other` group )

Reply:
Yes, `ArrayField` is binary data. Distinct from `ListField`